model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -182,13 +162,16 @@
     "\n",
     "# Information about deployed model.\n",
     "MODEL_NAMES = [\n",
-    "    'tf_gru_attention_unbiasN_trainY:v_1537919376',\n",
-    "    'tf_gru_attention_unbiasY_trainN:v_1537918796']\n",
+    "    'tf_gru_attention_civil:v_20181109_164318',\n",
```

Comment:
Can you add a comment here to say which model is which experiment (i.e. unbiasX? trainX?)